docs/stdlib/datetime.rst

```diff
@@ -461,29 +460,27 @@ EdgeDB stores and outputs timezone-aware values in UTC.
       ...        <cal::local_datetime>'2021-01-30T23:00:00' + m;
       {false}
 
-    See functions :eql:func:`cal::to_relative_duration`, and :eql:func:`to_str`
-    and date/time :eql:op:`operators <dtminus>` for more ways of working with
-    :eql:type:`cal::relative_duration`.
+    For more information regarding interacting with this type, see
+    :eql:func:`cal::to_relative_duration`, :eql:func:`to_str` and
+    :eql:op:`operators <dtminus>`.
 
 
 ----------
 
 
 .. eql:type:: cal::date_duration
 
-    A type representing a span of time in days.
+    Represents a timespan of a date.
```

Comment:
```suggestion
    A type for representing a timespan in days.
```

I don't think the rewrite is accurate. What's important here is that the granularity of the duration is limited.